pkgs/by-name/se/serie/package.nix

```diff
@@ -27,8 +30,9 @@ rustPlatform.buildRustPackage rec {
     ]
   );
 
-  # requires a git repository
-  doCheck = false;
+  nativeBuildInputs = [ gitMinimal ];
```

Comment:
Maybe put this in `nativeCheckInputs`?

Reply:
so you're saying rely on the git on the system for runtime and only have this as a check? the program fails to run correctly if git is not installed.

Reply:
`nativeCheckInputs` and `nativeBuildInputs` are pretty much the same thing but `nativeCheckInputs` adds stuff for when you are doing checks instead of the whole build.

Reply:
> nativeCheckInputs and nativeBuildInputs are pretty much the same thing but nativeCheckInputs adds stuff for when you are doing checks instead of the whole build.
I understand that but this *does* require git for runtime.

Reply:
For the entire build and not just checks?

Reply:
> For the entire build and not just checks?

for end user runtime and just checks

Reply:
Well, putting it in `nativeBuildInputs` still wouldn't fix for the runtime. Might have to wrap it or does it rely on the user already having git in `$PATH`? If so, still put it in `nativeCheckInputs`.

Reply:
> Well, putting it in nativeBuildInputs still wouldn't fix for the runtime. Might have to wrap it or does it rely on the user already having git in $PATH? If so, still put it in nativeCheckInputs.

it relies on git being in the path, maybe I should add add it to the description ?
with that understood ill switch over to native check inputs